proposals/NNNN-swiftpm-per-target-swift-language-version-setting.md

```diff
@@ -0,0 +1,54 @@
+# Swift Language Version Per Target
+
+* Proposal: [SE-NNNN](NNNN-swiftpm-per-target-swift-language-version-setting.md)
+* Authors: [Pavel Yaskevich](https://github.com/xedin)
+* Review Manager: TBD
+* Status: **Awaiting Review**
+* Implementation: https://github.com/apple/swift-package-manager/pull/7439
```

Comment:
```suggestion
* Implementation: [apple/swift-package-manager#7439](https://github.com/apple/swift-package-manager/pull/7439)
```

Use an explicit link so the value will appear on the SE Dashboard.

Reply:
Done!